cmd/get/util.go

```diff
@@ -0,0 +1,75 @@
+package get
+
+import (
+	"github.com/flyteorg/flytectl/cmd/config"
+	"github.com/flyteorg/flytectl/pkg/filters"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+func buildResourceListRequestWithName(c *config.Config, name string) *admin.ResourceListRequest {
+	fieldSelector, err := filters.Transform(filters.SplitTerms(c.FieldSelector))
+	if err != nil {
```

Comment:
if this fails do we ever bubble that up to the end user?

Reply:
no, We just discard the field selector and use plan get all query